testing/test_runners/all_tests.c

```diff
@@ -0,0 +1,11 @@
+#include "unity_fixture.h"
+
+static void RunAllTests(void)
+{
+  RUN_TEST_GROUP(LibraryCode);
```

Comment:
All of these files should follow the same formatting style as the rest of the code. (This one has wrong indentation.) (Unfortunately, `make format` won't work on them until we set up the `Makefile`, so don't worry about it yet.)